distribution/build.gradle

```diff
@@ -331,6 +331,7 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
 
   dependencies {
     libs project(':server')
+    libs project(':libs:swisshash')
```

Comment:
@rjernst I think that this is the first submodule that will be in the boot loader, but server does not depend on it. I added it here, as ultimately we will want to use it in other places beyond esql.